src/components/UserInfo.js

```diff
@@ -1,16 +1,33 @@
 export default class UserInfo {
-  constructor(nameSelector, professionSelector) {
-    this._name = document.querySelector(nameSelector);
-    this._profession = document.querySelector(professionSelector);
+  constructor(profileNameSelector, profileProfessionSelector, profileImageSelector) {
+    this._name = document.querySelector(profileNameSelector);
+    this._profession = document.querySelector(profileProfessionSelector);
+    this._avatarEl = document.querySelector(profileImageSelector);
   }
   getUserInfo() {
     return {
       name: this._name.textContent,
       profession: this._profession.textContent,
     };
   }
-  setUserInfo(name, profession) {
-    this._name.textContent = name;
-    this._profession.textContent = profession;
+
+  setUserInfo({ userData }) {
+    
+;    this._name.textContent = userData.name;
```

Comment:
You need to format the markup to show indentation and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`

Reply:
Reformatted with prettier